contracts/extensions/ede008-funded-proposal-submission-v2.clar

```diff
@@ -104,7 +105,7 @@
 
 ;; Proposals
 
-(define-public (fund (proposal <proposal-trait>) (amount uint))
+(define-public (fund (proposal <proposal-trait>) (amount uint) (custom-majority (optional uint)))
```

Comment:
I think we should not allow the public do set a custom majority. It is always in a user's interest to set the lowest value.

Rather, different extensions can set different levels.

We could create a test contract if the purpose of the parameter is to test the functionality.

Reply:
Ok! I'll leave the ede008-funded-proposal-submission-v2.clar as is (with per proposal custom majority) so we can play with it on testnet. We can change this to setting it at the extension level on mainnet deployment.